cmd/brew-test-bot.rb

```diff
@@ -1462,15 +1462,6 @@ def test_ci_upload(tap)
       "$GIT_TAG"
     end
 
-    if git_tag
-      if ARGV.include?("--dry-run")
-        puts "git push --force #{remote} master:master :refs/tags/#{git_tag}"
-      else
-        safe_system "git", "push", "--force", remote,
```

Comment:
This is still needed code to delete the existing tag. I'd suggest you add a `safe_system "git", "reset", "--hard", "origin/master"` before that.

Reply:
like this a169e54?